pkg/commands/loading_commits.go

```diff
@@ -312,14 +312,23 @@ func (c *CommitListBuilder) setCommitMergedStatuses(refName string, commits []*m
 }
 
 func (c *CommitListBuilder) getMergeBase(refName string) (string, error) {
+	baseBranch := c.GitCommand.TryGetMainBranch()
+	if baseBranch == "" {
+		baseBranch = "master"
+	}
+
 	currentBranch, _, err := c.GitCommand.CurrentBranchName()
 	if err != nil {
 		return "", err
 	}
-
-	baseBranch := "master"
-	if strings.HasPrefix(currentBranch, "feature/") {
-		baseBranch = "develop"
+	currentBranchParts := strings.SplitN(strings.ToLower(currentBranch), "/", 2)
+	if len(currentBranchParts) == 2 {
+		switch currentBranchParts[0] {
+		case "bugfix", "feature", "hotfix", "release":
```

Comment:
I think we should only do this for when `currentBranchParts[0]` is 'feature' because the others should be based on the master branch

Reply:
Fixed